app/controllers/application_controller.rb

```diff
@@ -22,9 +24,24 @@ class ApplicationController < ActionController::Base
      render_404 exception
   end
 
+  def profile
```

Comment:
On a dev machine, anywhere in the app, add a query param of ```trace=1```.

So for example if you want to generate a profile for http://localhost:3000/dashboard/files, you would visit http://localhost:3000/dashboard/files?trace=1. The page will (very slowly) load, and once it's done there will be a file called ```profile-stack.html``` in hydranorth folder with a graph of the profiler's results.

I'll update the handbook with this information too.